tests/Endpoints/IndexTest.php

```
@@ -244,37 +234,33 @@
+        self::assertSame(TaskType::IndexDeletion, $task->getType());
+
+        $task = $index->delete();
+        self::assertSame($indexName2, $task->getIndexUid());
+        self::assertSame(TaskType::IndexDeletion, $task->getType());
     }
 
     public function testSwapIndexes(): void
     {
-        $promise = $this->client->swapIndexes([['indexA', 'indexB'], ['indexC', 'indexD']]);
-        $response = $this->client->waitForTask($promise['taskUid']);
+        $task = $this->client->swapIndexes([['indexA', 'indexB'], ['indexC', 'indexD']])->wait();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$task'. (undefined)

(UnusedLocalVariable)